modules/bookCollection.js

```diff
@@ -0,0 +1,29 @@
+export default class BookCollection {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook(cTitle, cAuthor) {
+    const newBook = {
+      title: cTitle,
+      author: cAuthor,
+    };
+    this.books.push(newBook);
+    localStorage.setItem('localStorageBooks', JSON.stringify(this.books));
+  }
+
+  removeBook(index) {
+    const result = this.books.filter((book, i) => i !== index);
+    this.books = result;
+    localStorage.setItem('localStorageBooks', JSON.stringify(this.books));
+    window.location.reload();
+  }
+
+  getBooks() {
+    return this.books;
+  }
+
+  setBooks(books) {
+    this.books = books;
+  }
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=Refactor%20your%20methods%20and%20functions%20to%20arrow%20functions.), Please replace these methods with `ES6` arrow function syntax.
[Hint] For example:
```javascript
 addBook = (cTitle, cAuthor) => {
   // functionality here
}
```